packages/dds/test-dds-utils/src/test/ddsFuzzHarness.spec.ts

```diff
@@ -2,15 +2,18 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
This seems suspicious... it implies that typescript wasn't finding types for some of the APIs used here.

Reply:
It appears that I had misconfigured the exports for the test files.

I noticed that the utils.d.ts file wasn't being published to either the lib or dist folder. After adding `declaration: true` to the tsconfig.json eslint errors were resolved locally.